src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react'
+import { getArticles } from '../api'
+import ArticlePreview from './ArticlePreview';
+
+const ArticleList = () => {
+    const [articles, setArticles] = useState([]); 
+    const [isLoading, setIsLoading] = useState(true); 
+
+    useEffect(() => {
+        getArticles()
+        .then(({ data }) => {
+            setArticles(data)
+            setIsLoading(false); 
+        })
+    }, [])
+
+   if (isLoading) return  <p>Loading!</p>
+
+    const formatArticles = articles.map((article) => {
+        return (
+            <ArticlePreview key={article.title} article={article}/>
```

Comment:
Article id would be more reliable as a key here! What if two articles had the same title?